src/Tester/Tester.csproj

```diff
@@ -43,62 +43,67 @@
     <AssemblyOriginatorKeyFile>..\Orleans.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Bond">
-      <HintPath>$(SolutionDir)packages\Bond.Runtime.CSharp.3.0.7\lib\net45\Bond.dll</HintPath>
```

Comment:
>The corollary is also true - if a particular changed line is not directly related to the PR change in hand, then it really should be split out into a separate PR / change set.
You will frequently see @gabikliot say exactly that during code review: "please submit [this part] as a separate PR"
By keeping change sets small and focused, it is much easier to isolate any problems that may be introduced along the way, even if later on we have to go back and exclude change requests one-by-one to see exactly what caused the issue to start.

Yes! I totally and fully support that.
I would even add: by making separate PRs for unrelated features, you are making the life of the reviewer much easier. It is his responsibility to fully understand and approve a PR, therefore by making smaller PRs you increase your chances for a PR to be accepted and the product be more robust.